.github/actions/approval-test/action.yml

```diff
@@ -0,0 +1,24 @@
+# Expects ./scripts/install_build_dependencies.sh to have been ran prior to
+# running steps of this file.
+
+name: 'C encapsulation approval test'
+description: 'Run tests for verifying expected program behavior'
```

Comment:
Please remove work 'program'. Testing build is not testing program behavior, specifically.